templates/csi-driver-object.yaml.j2

```diff
@@ -3,5 +3,5 @@ kind: CSIDriver
 metadata:
   name: kadalu
 spec:
-  attachRequired: true
+  attachRequired: false
```

Comment:
don't change this or else existing deployments on operator restart will fail for k8s 1.14 to 1.21

Reply:
Ack